src/Common/CommonAssemblyInfo.cs

```diff
@@ -26,8 +26,8 @@
 
 #if ASPNETWEBHOOKS
 #if !BUILD_GENERATED_VERSION
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyFileVersion("2.0.0.0")]
```

Comment:
This file is used in the old-world projects. For now, they won't have the same assembly attributes e.g. no `[assembly: AssemblyInformationalVersion("2.0.0-preview1-t000")]` or `[assembly: AssemblyMetadata("BuildNumber", "t000")]`.